ext_mongodb.php

```diff
@@ -19,6 +19,108 @@ function bsonUnserialize(array $data) : void;
 {
 }
 
+/* == MONITORING ============================================================= */
+namespace MongoDB\Driver\Monitoring;
+
+/* ------ Building blocks for value obejcts */
+abstract class CommandEvent
+{
+	private function __construct()
+	{
+		throw new Exception\RunTimeException("Accessing private constructor");
+	}
+
+	public function getCommandName() : string;
+
+	// do we need this? I don't think so, as you pass in an object
+	public function getContext() : mixed;
+
+	// libmongoc use getHost, but spec says "connectionId".
+	public function getHost() : \MongoDB\Driver\Server;
+
+	// Not adding getHint, as we don't really expose that.
```

Comment:
A "hint" in libmongoc is basically our Server object, although I suppose our Server is really a "hint" that was resolved to a `mongoc_server_description_t` or `mongoc_host_list_t`. Anyway, I agree there is nothing to handle here.